python/ray/monitor.py

```diff
@@ -478,6 +481,103 @@ def driver_removed_handler(self, unused_channel, data):
 
         self._clean_up_entries_for_driver(driver_id)
 
+    def _xray_clean_up_entries_for_driver(self, driver_id):
+        """Remove this driver's object/task entries from redis.
+
+        Removes control-state entries of all tasks and task return
+        objects belonging to the driver.
+
+        Args:
+            driver_id: The driver id.
+        """
+        redis = self.state.redis_clients[0]
+
+        XRAY_TASK_TABLE_PREFIX = ray.gcs_utils.TablePrefix_RAYLET_TASK_string\
```

Comment:
We should really enforce this in our linting, but let's avoid using `\` and instead write this like 

```python
XRAY_TASK_TABLE_PREFIX = ray.gcs_utils.TablePrefix_RAYLET_TASK_string.encode(
    "ascii")

# or 

XRAY_TASK_TABLE_PREFIX = (
    ray.gcs_utils.TablePrefix_RAYLET_TASK_string.encode("ascii"))
```

Reply:
Let's avoid using all caps for temporary variables